libs/model/src/main/kotlin/de/cyface/model/RequestMetaData.kt

```diff
@@ -101,17 +102,18 @@ data class RequestMetaData(
         require(imageCount >= 0) { "Invalid imageCount: $imageCount" }
         require(videoCount >= 0) { "Invalid videoCount: $videoCount" }
         require(filesSize >= 0) { "Invalid filesSize: $filesSize" }
+        require(attachmentIdentifier == null || attachmentIdentifier.length <= MAX_ID_LENGTH) {
+            "Field attachmentId had an invalid length of ${attachmentIdentifier!!.length.toLong()}"
```

Comment:
This is going to crash hard if attachmentIdentifier is actually null. Using !! is also usually a Code smell.